### ISSUE TITLE
Unable to select item

### ISSUE THREAD

**Issue by bennypowers** [GoogleWebComponents/google-chart#199]
I have a timeline chart:

```html
<google-chart id="chart"
    type="timeline"
    cols="[[cols]]"
    rows="[[rows]]"
    selection="{{chartSelection}}"
    options="[[options]]"
></google-chart>
```

And a method:

```js
_activeClipChanged(activeClip, activeClipset) {
  if (!activeClipset) return;
  const row = activeClipset.indexOf(activeClip);
  this.set('chartSelection', [{row, column: null}]);
},
```

And the two are not getting along. I can't seem to  set the selection in the google chart from outside.

Reproducable JSBin: https://jsbin.com/qukucon/edit?html,output

**Comment by wesalvaro** [comment]
Thanks so much for the jsbin!
I've tracked down the docs and responsible code.
`setSelection` is actually, currently unavailable for the Timeline chart.
While the method exists and does update the chart state, it does not seem possible to draw the chart with the selection set by the user. I've raised a bug for this internally.
But a better place to file this would be with the Google Visualization team: https://github.com/google/google-visualization-issues/issues
